rules/templatetags/rules.py

```diff
@@ -5,13 +5,19 @@
 
 register = template.Library()
 
+try:
+    assignment_tag = register.assignment_tag
+except AttributeError:
+    # django < 2.0
+    assignment_tag = register.simple_tag
```

Comment:
I think you got this the opposite way round? `register.assignment_tag` is Django < 2.0.